app/Transformers/Api/Client/DatabaseTransformer.php

```diff
@@ -21,6 +21,7 @@ public function transform(Database $model): array
         $model->loadMissing('host');
 
         return [
+            'driver' => $model->getRelation('host')->driver,
             'id' => $model->id,
             'host' => [
                 'address' => $model->getRelation('host')->host,
```

Comment:
Not your fault, but should quick fix this nonetheless:

```suggestion
            'driver' => $model->host->driver,
            'id' => $model->id,
            'host' => [
                'address' => $model->host->host,
```